Cargo.toml

```diff
@@ -24,11 +25,7 @@ members = [
     "swayfmt",
     "test",
 ]
-exclude = [
-    "examples/*",
-    "swayfmt/test_macros",
-    "forc-test/test_data"
-]
+exclude = ["examples/*", "swayfmt/test_macros", "forc-test/test_data"]
```

Comment:
Not sure how we have this change, did `rustfmt` version changed?

Reply:
It was my toml fmt. I'll revert this.